samples/daily-prebuilt/permissions-can-admin/src/daily.js

```diff
@@ -0,0 +1,39 @@
+const createRoom = async () => {
+  const roomRes = await fetch('/api/rooms', {
+    method: 'POST',
+    body: JSON.stringify({
+      properties: {
+        enable_prejoin_ui: false,
```

Comment:
One other thing I just realized - let's make sure to add an expiry to this room!